City-of-Helsinki/parking-permits
# Parking permit expiration remind email

## Description

We need to send a permit expiration remind-email for fixed-period permit one week before permit validity expiration day.

## Context

Refs: [PV-388](https://helsinkisolutionoffice.atlassian.net/browse/PV-388)

## How Has This Been Tested?

Manually, by running Django management command `./manage.py remind_permit_expiration` that will send remind-emails automatically to users with a permit, that will expire within a week from now.

## Manual Testing Instructions for Reviewers

Run Django management command `./manage.py remind_permit_expiration` and observe the application console (easiest test case without local email-setup).
